### PR TITLE
feat: LT-1413 Default Queue

### PR DESCRIPTION
**Pull request by andytaylorkainos**

### CODE REVIEW

File: applications/forms/common.py [hunk 1]
```diff
@@ -232,7 +232,9 @@ def declaration_form(application_id):
                 title=strings.declaration.Declaration.RADIO_TITLE,
                 options=[
                     Option(
-                        "True", strings.declaration.Declaration.AGREE_TO_DECLARATION, test_id="agreed_to_declaration"
+                        "True",
```
Comment (BenAndrew-Kainos) [inline applications/forms/common.py:235]: Jan says we don't put quotes around the key if it's a boolean

Comment (BenAndrew-Kainos) [reply applications/forms/common.py:235]: also why did this file change? Is your black setup differently?

Comment (andytaylorkainos) [reply applications/forms/common.py:235]: data_attribute="agreed_to_declaration" is the change

Comment (jfaracik) [reply applications/forms/common.py:235]: Yeah, is converting to a string necessary?

**Pull request merged**